src/mixins/themeable.ts

```diff
@@ -74,25 +74,16 @@ export interface ThemeableMixin extends Evented {
  * Themeable
  */
 export interface Themeable extends ThemeableMixin {
-	baseClasses: BaseClasses;
+	baseClasses: {};
```

Comment:
It does not work for me when i create a custom widget and try to set it's imported CSS as `baseClasses`. I get the message: